README.md

```diff
@@ -1,7 +1,7 @@
 # ![Crayfish](https://cloud.githubusercontent.com/assets/2371345/15409657/2dfb463a-1dec-11e6-9089-06df94ef3f37.png) Crayfish
 
 [![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%205.6-8892BF.svg?style=flat-square)](https://php.net/)
-[![Build Status](https://travis-ci.org/Islandora-CLAW/Crayfish.svg?branch=master)](https://travis-ci.org/Islandora-CLAW/Crayfish)
+[![Build Status](https://travis-ci.org/Islandora-CLAW/Crayfish.svg?branch=master)](https://travis-ci.com/Islandora-CLAW/Crayfish)
```

Comment:
The first reference to Travis-CI still points to .org

Reply:
thanks... i need to touch up all the other PRs i've got attempting the same :man_facepalming: